packages/web-app-files/src/views/Personal.vue

```diff
@@ -220,84 +217,36 @@ export default defineComponent({
 
   methods: {
     ...mapActions('Files', ['loadPreview']),
-    ...mapActions(['showMessage']),
+    ...mapActions(['showMessage', 'createModal', 'hideModal']),
     ...mapMutations('Files', ['REMOVE_FILE', 'REMOVE_FILE_FROM_SEARCHED', 'REMOVE_FILE_SELECTION']),
 
-    fetchResources,
+    async fetchResources(path, properties) {
+      try {
+        return await this.$client.files.list(path, 1, properties)
```

Comment:
`fetchResources` comes from the folder loader service. What's the reason for switching to a custom implementation here? only for development and to be reverted?

Reply:
didnt know better to be honest, will try it out 👍🏼